### PR TITLE
Feat/embed update

### PR DESCRIPTION
**Pull request by jepser**
- Updated embed script
- Refactor shortcode to avoid loading js/css for each typeform and use Shareinguito implementation instead
- Remove iframe and use Shareinguito implementation
- Some renaming

### CODE REVIEW

File: parts/typeform-embed.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<div
+	class="typeform-widget"
+	data-url="<?= $url; ?>"
+	style="width: <?= $width; ?>; height: <?= $height; ?>;"
```
Comment (alexbassy) [inline parts/typeform-embed.php]: Shouldn't these `<?= literals ?>` be on separate lines?

File: parts/typeform-popup.php [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<?php
+    $style = ($style) ? $style: 'link';
+	$type = ($type == 'drawer') ? 2: 1;
```
Comment (alexbassy) [inline parts/typeform-popup.php]: Could you avoid the reassignment of the variable here?

File: parts/typeform-popup.php [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<?php
+    $style = ($style) ? $style: 'link';
```
Comment (alexbassy) [inline parts/typeform-popup.php]: Could you add a space before the `:`? It kinda looks like a `key: value` at first glance

File: typeform-shortcodes.php [hunk 4]
```diff
@@ -1,39 +1,71 @@
...
+        $options['url'] = apply_filters('typeform_embed_url', $options['url'], $options['builder']);
+
+        return (object) $options;
     }
 
-    if (strpos($width, '%') === false && strpos($width, 'px') === false) {
-        $width = (string) $width . 'px';
+    static function isValidValue($value)
+    {
+        return strpos($value, '%') !== false || strpos($value, 'px') !== false;
```
Comment (alexbassy) [inline typeform-shortcodes.php]: is the `!== false` redundant here?

**Review by alexbassy — CHANGES_REQUESTED** [review]

File: parts/typeform-popup.php (hunk 2 above)
Comment (jepser) [reply parts/typeform-popup.php]: That's true, I didn't refactor this.

**Comment by alexbassy** [comment]
@jepser I still see the `$condition ? $one: $two` thing - is this intentional / your style? See [my comment](https://github.com/jepser/typeform/pull/7#discussion_r170984345)

File: parts/typeform-popup.php (hunk 3 above)
Comment (jepser) [reply parts/typeform-popup.php]: That's true! I will change it

**Review by alexbassy — APPROVED** [review]

**Comment by alexbassy** [comment]
👍

**Pull request merged**